app/web/src/components/Workspace/WorkspaceAdminDashboard.vue

```diff
@@ -82,4 +113,9 @@ const killExecution = () => {
 };
 
 const funcRunId = ref<string | null>(null);
+
+const featureFlags = computed(() => {
+  const flags = Object.entries(featureFlagStore.$state);
+  return flags.filter((flag) => flag[0] !== "apiRequestStatuses");
```

Comment:
Note: we could just export `FeatureFlags` ... or do something else to avoid iterating over `$state`